helix-term/src/commands.rs

```diff
@@ -2067,8 +2068,12 @@ fn global_search(cx: &mut Context) {
                     .binary_detection(BinaryDetection::quit(b'\x00'))
                     .build();
 
-                let search_root = std::env::current_dir()
-                    .expect("Global search error: Failed to get current dir");
+                let search_root = std::env::current_dir();
+                if search_root.is_err() {
+                    _editor.set_error("Global search error: Failed to get current dir");
+                    return;
+                }
+                let search_root = search_root.unwrap();
```

Comment:
This can be cleaned up a little:

```rust
let search_root = match std::env::current_dir() {
    Ok(cwd) => cwd,
    Err(err) => {
        editor.set_error(format!("Global search error: {err}"));
        return;
    }
};
```

`_editor` here should have the underscore removed, and in the closure's parameter list above. It was underscored previously since it was unused.